modules/pv.py

```diff
@@ -96,32 +111,61 @@ def pv_presence_check(self, disk):
         rc, out, err = self.run_command('pvdisplay', ' ' + disk)
         ret = 0
         if self.action == 'create' and not rc:
-            self.module.exit_json(rc=0, changed= 0, msg="%s Physical Volume Exists!" % disk)
+            self.module.exit_json(rc=0, changed= 0, msg="%s Physical Volume"
+                                   " Exists!" % disk)
         elif self.action == 'remove' and rc:
-            self.module.exit_json(rc=0, changed=0,msg="%s Physical Volume Doesn't Exists!" % disk)
+            self.module.exit_json(rc=0, changed=0,msg="%s Physical Volume"
+                                   "Doesn't Exists!" % disk)
         else:
             ret = 1
         return ret
 
     def pv_action(self):
-        self.disks = self.validated_params('disks')
+        self.disks = self.module.params['disk']
         if not self.disks:
-            self.module.exit_json(msg="Nothing to do")
+            self.disks = self.module.params['disks']
+            self.module.exit_json(msg='disks paramater is deprecated, please consider using disk', rc=0, changed=True)
```

Comment:
@ashmitha7 this will exit the pv creation, it should not exit. We just print a warning.